kubernetes/windows/main.ps1

```diff
@@ -35,6 +35,27 @@ function Remove-WindowsServiceIfItExists($name) {
     }
 }
 
+function Test-FluentbitTcpListener {
+    param (
+        [int]$port
+    )
+    $retryCount = 30
```

Comment:
does this mean now there can be a 30 second delay?

Reply:
> does this mean now there can be a 30 second delay?

update code to be consistent with linux agent.